### PR TITLE
Adding atom feed for blog

### PR DESCRIPTION
**Pull request by uidoyen**

### CODE REVIEW

File: _layouts/base.html [hunk 1]
```diff
@@ -7,6 +7,7 @@
   <meta charset="utf-8">
   <meta name="viewport" content="width=device-width, initial-scale=1">
   <meta name="description" content="{% if page.description %}{{ page.description }}{% else %}{{ site.description }}{% endif %}">
+  <link rel="alternate" type="application/rss+xml" title="Bakery Store RSS" href="/blog.atom">
```
Comment (gunnarmorling) [inline _layouts/base.html:10]: Bakery store ;) ?

Comment (uidoyen) [reply _layouts/base.html]: Sorry, my bad.

Comment (gunnarmorling) [reply _layouts/base.html]: No worries. I guess I'd find this on Stack Overflow ;) ?

**Comment by gunnarmorling** [comment]
Fixes #567.

**Pull request merged**